config/cps-sink-connector.properties

```diff
@@ -1,8 +1,32 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Unique name for the Pub/Sub sink connector.
 name=CPSSinkConnector
+# Tha Java class for the Pub/Sub sink connector.
 connector.class=com.google.pubsub.kafka.sink.CloudPubSubSinkConnector
+# The maximum number of tasks that should be created for this connector.
 tasks.max=10
-topics=my-kafka-topic
-cps.topic=my-cps-topic
-cps.project=my-cps-project
+# Set the key converter for the Pub/Sub sink connector.
 key.converter=org.apache.kafka.connect.storage.StringConverter
+# Set the value converter for the Pub/Sub sink connector.
 value.converter=org.apache.kafka.connect.converters.ByteArrayConverter
+# A comma-seperated list of Kafka topics to use as input for the connector.
+# TODO (developer): update to your Kafka topic name(s).
+topics=my-kafka-topic
+# TODO (developer): update to your GCP project ID.
```

Comment:
Can we add comments describing what the config parameter is apart from the TODO? Something like `# The Google Cloud project that owns the Cloud Pub/Sub topic you want to use as the destination for the sink connector.` for `cps.project`

Reply:
This is meant to for the quickstart in README, which has a prereq section that describes what projects/topics are.

More details for this  and other config options are also in the README. Keeping it brief is in line with what we do for all GCP samples. See https://cloud.google.com/pubsub/docs/publisher#publish_messages

Reply:
We can keep it brief but since we've added comments for the other fields I think we should be consistent,